Source Code/Front-End/UI (WPF)/Smart Pacifier - Tool/Smart Pacifier - Tool/Tabs/SettingsTab/SettingsView.xaml.cs

```diff
@@ -259,6 +262,7 @@ private void LocalButton_Click(object sender, RoutedEventArgs e)
         {
             LocalHostPanel.Visibility = Visibility.Visible;
             InfluxDbModePanel.Visibility = Visibility.Collapsed;
+            InfluxDbWebView.Visibility = Visibility.Hidden;

```

Comment:
This shouldn't be hidden, if the local instance of the DB is already running either activated in the App or beforehand, it would not visible to the user if they view the UI for the local interface.